tests/scripts/run_tests.py

```diff
@@ -26,7 +26,7 @@
 class Testn(TestCase):
     def decode(self, ccdl_path, data_path):
         with open(ccdl_path, 'r') as f:
-            my_types = cddl_gen.DataTranslator.from_cddl(f.read(), 16)
+            my_types, _ = cddl_gen.DataTranslator.from_cddl(f.read(), 16)
```

Comment:
It doesn't feel right to me to return a tuple that may only have one element of interest. Why not a `NamedTuple` with `types` and `control_groups`?

We would probably avoid updating a lot of calling code from API changes like this if we always returned a object, then modified its internals as needed for callers to use/ignore at their discretion.

Reply:
Changed to NamedTuple.